pkg/geo/geo.go

```diff
@@ -617,6 +618,37 @@ func S2RegionsFromGeom(geomRepr geom.T, emptyBehavior EmptyBehavior) ([]s2.Regio
 // Common
 //
 
+// Normalize geographical coordinates
+// to spherical coordinates
+func makeValidGeographicalPoint(lat float64, lng float64) (float64, float64) {
```

Comment:
let's name this `normalizeLngLat`, so we traverse in array order (going arr[i+1] then arr[i] i'm sure misses some CPU optimization magic).
make this comment one line, i.e. 'normalizeLngLat normalizes geographical coordinates into a valid range.`